app/Services/AdminService.php

```diff
@@ -13,4 +14,14 @@ public static function sendInvite($studentEmail)
 
         return "email has been sent";
     }
+
+    public function storeAdmin($data)
+    {
+        return UserInfo::create([
+            'user_id' =>  "1",
```

Comment:
Instead of hard coding the user_id, we can do a couple of things: 
- Use eloquent relations to figure out what the related user_id is. 
- Or to pass the user_id through the request. 

On the front end, they should have access to the current user's id so the second suggestion should be valid.